packages/client/hmi-client/src/workflow/ops/model-coupling/tera-model-coupling-node.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+	<main>Model coupling</main>
```

Comment:
You are missing the [tera-operator-placeholder](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/components/operator/tera-operator-placeholder.vue).